xbmc/pvr/dialogs/GUIDialogPVRChannelManager.cpp

```diff
@@ -705,11 +705,11 @@ void CGUIDialogPVRChannelManager::Update()
   if(!channels)
     return;
 
-  const std::vector<std::shared_ptr<PVRChannelGroupMember>> groupMembers = channels->GetMembers();
+  const std::vector<std::shared_ptr<CPVRChannelGroupMember>> groupMembers = channels->GetMembers();
```

Comment:
`const auto`?

Reply:
This PR was not meant to be a code cleanup PR. This should be done in a followup PR making this consistent for the whole PVR component. ;-)